azure/src/services/azureFunctionCloudService.test.ts

```diff
@@ -91,7 +110,13 @@ describe("Azure Cloud Service should", () => {
       data: payload
     };
 
-    const response = cloudService.invoke("azure-getCart", false, payload);
+    const params: InvokeRequest = {
+      name: "azure-getCart",
+      fireAndForget: false,
+      payload: payload,
+      headers: new StringParams()
+    };
+    const response = cloudService.invoke(params);
```

Comment:
Same as above about await statement

Reply:
Thanks for this feedback, it will be added in my next commit